recipes/pandera-core/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pandera" %}
+{% set version = "0.4.5" %}
+
+package:
+  name: {{ name|lower }}-core
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb0efb6e8d7d1a5edfbf15d8e8c8c0958c27484a98cffcd371ef688707b0b19c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - autopep8
+    - black
+    - numpy
+    - pandas
+    - pyyaml
+    - scipy
+    - wrapt
+
+test:
+  imports:
+    - pandera
```

Can we try to add ``pip check`` here as well please?
```suggestion
    - pandera
  requires:
    - pip
  commands:
    - pip check
```